CONTRIBUTORS.md

```diff
@@ -0,0 +1,112 @@
+# Outline
+
+1. Overview
+2. Developing a Contribution
+3. Pre-screening a Contribution
+4. Submitting a Contribution
+5. Acceptance Criteria
+6. From Contribution to Commit
+7. Rationale
+
+# Overview
+
+The life-cycle of a contribution is as follows:
+
+1. Contributor improves or expands the code-base and wants to contribute the change to the community project.
+2. Contributor pre-screens the contribution against the Acceptance Criteria.
+3. Contributor submits the contribution to the Community for review.
+4. The Community reviews the contribution against Acceptance Criteria.
+5. The Contributor works with the Community to update the contribution until it meets all criteria.
+6. The Committers sign-off on the contribution and merge it into the code-base.
+
+Note: The term 'Community' is intentionally used to highlight that Contributors, Committers, PMC, or anyone else, can be a part of the process. 
+
+# Developing a Contribution
+
+If you are a complete beginner to Apache NuttX, please refer to our Getting Started Guide [Add link]. This will help you get up and running with the stock project.
```

Comment:
```suggestion
If you are a complete beginner to Apache NuttX, please refer to our Getting Started Guide [Add link]. This will help you get up and running with the Apache NuttX repositories
```